Sources/AnyLint/Statistics.swift

```diff
@@ -2,7 +2,7 @@
 import Utility
 
 final class Statistics {
-    static let shared = Statistics()
+    static let `default` = Statistics()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Properties should have a type interface](https://app.codacy.com/gh/Flinesoft/AnyLint/pullRequest?prid=5691372)